src/wasmJsMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/system.wasm.kt

```diff
@@ -1,5 +1,9 @@
 package it.krzeminski.snakeyaml.engine.kmp.internal
 
+internal actual fun getEnvironmentVariable(key: String): String? {
+    TODO()
+}
```

Comment:
issue: this causes `Conflicting overloads` error - we have same method in `jsCommon` sources